Dockerfile-sandbox

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ARG sdk_version=0.13.22
+
+FROM digitalasset/daml-sdk:${sdk_version}
+
+EXPOSE 6865
```

Comment:
OK, let's move this line back down to just before the `CMD` so we can share the cache with _Dockerfile-navigator_.

Reply:
i tried a few times and think it is leveraging caching now